README.md

```diff
@@ -24,7 +24,7 @@ The [`vscode`](https://github.com/microsoft/vscode) repository is where we do de
 ## Contributing
 
 If you are interested in fixing issues and contributing directly to the code base,
-please see the document [How to Contribute](https://github.com/Microsoft/vscode/wiki/How-to-Contribute), which covers the following:
+please see the document here [How to Contribute](https://github.com/Microsoft/vscode/wiki/How-to-Contribute), which covers the following:
```

Comment:
Adding the "here" is uneccessary, "... see the document How to Contribute, which covers ..." gets to the point quick and easy.